index.js

```diff
@@ -30,11 +31,9 @@ function NYC (opts) {
   config = config.nyc || {}
 
   // load exclude stanza from config.
-  this.exclude = config.exclude || ['node_modules[\/\\\\]', 'test[\/\\\\]', 'test\\.js']
+  this.exclude = ['node_modules/**'].concat(config.exclude || ['test/**', 'test.js'])
```

Comment:
I would ignore `test-*.js` by default too. It's a very common pattern. https://github.com/sindresorhus/ava/blob/eaf3e0f8a66980e87496a30b9f4a8c45329f0ae9/api.js#L198